compiler/src/utils/wasm_utils.re

```diff
@@ -520,14 +534,6 @@ let () =
       )
     | [@implicit_arity] MalformedSectionType(tag, None) =>
       Some(Printf.sprintf("Malformed WASM section tag: %d", tag))
-    | [@implicit_arity] Wasm.Script.Syntax(src, msg) =>
-      Some(
-        Printf.sprintf(
-          "WebAssembly Parse Error at %s: %s",
-          Wasm.Source.string_of_region(src),
-          msg,
-        ),
-      )
```

Comment:
We can't raise this error anymore, since we don't parse wasm files using the wasm spec.

Reply:
Is that for the better or worse?

Reply:
It's a good thing! Pre-Binaryen, one of the ways we validated modules was by creating the bytes and then parsing it back to make sure it was correct. Now we just assume it's correct since it was produced via Binaryen :P